src/pybind/mgr/dashboard/frontend/src/app/core/auth/logout/logout.component.scss

```diff
@@ -0,0 +1,21 @@
+.logout-class {
```

Comment:
Is this a style defined somewhere else? If that is the case, could we use `@extend` to clone that style instead of redefining it? Just a question, I'm a ?CSS newbie.